### PR TITLE
feat: Implement V0 of docker import

### PR DESCRIPTION
**Pull request by victorcolombo**
## Description:

This PR implements the `kurtosis import` command in the CLI, that parses a very simple Docker Compose into Starlark and Runs it. Feature set is limited to:
- Image
- Ports
- Entrypoint / Command

Notably does not include:
- Volumes
- Env Vars
## Is this change user facing?
YES

### CODE REVIEW

File: cli/cli/commands/import/import.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+		serviceStarlarks[serviceConfig.Name] = starlarkConfig
+	}
+	script := "def run(plan):\n"
+	for serviceName, serviceConfig := range serviceStarlarks {
+		script += fmt.Sprintf("\tplan.add_service(name = '%s', config = %s)\n", serviceName, serviceConfig)
+	}
+	return script, nil
+}
+
+func runStarlark(ctx context.Context, starlarkScript string) error {
```
Comment (h4ck3rk3y) [inline cli/cli/commands/import/import.go]: perhaps we can refactor this later

File: cli/cli/commands/import/import.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	kurtosisCtx, err := kurtosis_context.NewKurtosisContextFromLocalEngine()
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred connecting to the local Kurtosis engine")
+	}
+
+	enclaveCtx, err := kurtosisCtx.CreateEnclave(ctx, "docker-compose-import", false)
```
Comment (h4ck3rk3y) [inline cli/cli/commands/import/import.go]: is this a prefix? If you look at `run` you can see how a new one is generated

File: cli/cli/commands/import/import.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred creating an enclave '%v'", "docker-compose-import")
```
Comment (h4ck3rk3y) [inline cli/cli/commands/import/import.go]: let's change this when we change the above

File: cli/cli/commands/import/import.go [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+	}
+
+	starlarkRunResult, err := enclaveCtx.RunStarlarkScriptBlocking(ctx, "", starlarkScript, "{}", false, 1, []kurtosis_core_rpc_api_bindings.KurtosisFeatureFlag{})
```
Comment (h4ck3rk3y) [inline cli/cli/commands/import/import.go]: lets put these constants in variables

File: cli/cli/commands/import/import.go [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+	if err != nil {
+		return stacktrace.Propagate(err, "An error has occurred when running Starlark to add service")
+	}
+	logrus.Infof("Enclave was built with following output: %s", starlarkRunResult.RunOutput)
```
Comment (h4ck3rk3y) [inline cli/cli/commands/import/import.go]: can we call inspect at the end like we do from `run`

File: cli/cli/commands/import/import.go [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+	if err != nil {
+		return "", stacktrace.Propagate(err, "Error translating docker compose to Starlark")
+	}
+	return script, nil
+}
+
+// TODO(victor.colombo): Have a better UX letting people know ports have been remapped
+func convertComposeProjectToStarlark(compose *types.Project) (string, error) {
+	serviceStarlarks := map[string]string{}
+	for _, serviceConfig := range compose.Services {
```
Comment (h4ck3rk3y) [inline cli/cli/commands/import/import.go:128]: perhaps we can track TODO for supporting `depends` later on

**Review by h4ck3rk3y — APPROVED** [review]

File: cli/cli/commands/import/import.go (hunk 1 above)
Comment (victorcolombo) [reply cli/cli/commands/import/import.go]: I'll add a TODO

File: cli/cli/commands/import/import.go (hunk 2 above)
Comment (victorcolombo) [reply cli/cli/commands/import/import.go]: Added a flag to autogenerate/set enclave name

File: cli/cli/commands/import/import.go (hunk 4 above)
Comment (victorcolombo) [reply cli/cli/commands/import/import.go]: Done

File: cli/cli/commands/import/import.go (hunk 5 above)
Comment (victorcolombo) [reply cli/cli/commands/import/import.go]: Added enclave name to facilitate ls later

File: cli/cli/commands/import/import.go (hunk 3 above)
Comment (victorcolombo) [reply cli/cli/commands/import/import.go]: Done

**Pull request merged**